src/main/java/core/basesyntax/io/write/impl/csv/CsvWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.io.write.impl.csv;
+
+import core.basesyntax.io.write.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvWriterImpl implements ReportWriter {
+    @Override
+    public void write(String report, String reportPath) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(reportPath))) {
+            bufferedWriter.write(reportPath);
```

Comment:
It seems like you're writing the `reportPath` to the file instead of the actual `report` content. You should write the content of the `report` parameter to the file.